src/injector/inject.ts

```diff
@@ -49,38 +48,64 @@ function getExtensionIdForOpcode (opcode: string) {
  * @param vm Virtual machine instance. For some reasons we cannot use VM here.
  * @returns Blockly instance.
  */
-function getBlocklyInstance (vm: EurekaCompatibleVM): any | null {
-    // Hijack Function.prototype.apply to get React element instance.
-    function hijack (fn: (...args: unknown[]) => unknown) {
-        const _orig = Function.prototype.apply;
-        Function.prototype.apply = function (thisArg: any) {
-            return thisArg;
-        };
-        const result = fn();
-        Function.prototype.apply = _orig;
-        return result;
-    }
+async function getBlocklyInstance (vm: EurekaCompatibleVM): Promise<any> {
```

Comment:
where is timeout?

Reply:
see `setTimeout` below

Reply:
timeout is not only necessary for alternative method, but also for this

Reply:
`initalized` flag should work. If timeout exceeded, `initalized` flag will be set to `true` and then `getBlocklyInstance` callback will be aborted.